src/client/src/apps/MainRoute/widgets/workspace/selectors.ts

```diff
@@ -17,16 +17,18 @@ const makeExternalWindowProps: (key: string, tileLayout?: WindowPosition) => Ext
   config: {
     name: `${key}`,
     width: 366, // 346 content + 10 padding
-    height: 202,
+    height: 208,
     minWidth: 366,
-    minHeight: 202,
-    maxWidth: 366,
-    maxHeight: 202,
+    minHeight: 208,
+    // TODO: If we are able to disable dropping other windows
+    // on to the SpotTiles, we can re-enable the maxWidth and maxHeight properties
+    // maxWidth: 366,
+    // maxHeight: 208,
```

Comment:
The styling of spot tiles under OpenFin Platform API shouldn't be messing with the styling of the normal web app.